tools/dashboard/src/clj/dashboard/main.clj

```diff
@@ -34,7 +35,11 @@
             {:interval 10 :key :jenkins
              :task #'data-jenkins/jobs}
             {:interval 60 :key :db-load
-             :task #'data-cloudwatch/finap-db-load}])
+             :task #'data-cloudwatch/finap-db-load}
+            {:interval (* 60 60 3) :key :sprint-themes
```

Comment:
There is quite many magic numbers. Not sure if it helps to understand this, but those could be moved to somewhere where they are explained.